Update Glossary (#387)
zkSNACKs/WasabiDoc: docs/glossary/README.md

:::

:::details
### Fungibility

> Fungibility is a property of Bitcoin UTXOs that are indistinguishable from each other.

Reviewer: ```suggestion
Fungibility is a desirable property of Bitcoin UTXOs that are indistinguishable from each other.
```

> Fungibility is a property of a Bitcoin utxo which are indistinguishable from each other.

Reviewer 2: @MaxHillebrand it contradicts this:
`However, an unspent transaction output, a bitcoin, is a chunk of money and is not fungible` https://docs.wasabiwallet.io/using-wasabi/Receive.html#the-importance-of-labeling

Reviewer: Well, the goal of Bitcoin privacy is to have fungibility in UTXOs [sats are already fungible].

So a non-fungible UTXO is tied to your KYC data.
a fungible UTXO is CoinJoined [or other tools] so it's equal among peers [anonset]

> Fungibility is a property of a Bitcoin utxo which are indistinguishable from each other.

Reviewer 2: ```suggestion
Fungibility is a property of Bitcoin UTXOs that are indistinguishable from each other.
```

> Fungibility is a (missing) property of a bitcoin (or satoshis), which should make them indistinguishable from each other.

Reviewer: satoshis are fungible - utxos are not.

```suggestion
Fungibility is a property of a Bitcoin utxo which are indistinguishable from each other.
```

> Read more: [What fee should I select?](/why-wasabi/TransactionSurveillanceCompanies.md#attempt-to-destroy-fungibility)

Reviewer: ```suggestion
Read more: [Transaction surveillance companies attempting to destroy fungibility](/why-wasabi/TransactionSurveillanceCompanies.md#attempt-to-destroy-fungibility)
```

Read more: [What is the anonimity set?](/FAQ/FAQ-UseWasabi.md#what-is-the-anonymity-set)
:::

:::details
### Bloom Filter

> A filter used primarily by SPV clients to request only matching transactions and merkle blocks from full nodes.

Reviewer: ```suggestion
A filter used primarily by SPV clients to request only block headers and merkle proofs of a given transaction from full nodes.
```

:::

:::details
### Know Your Customer (KYC)

KYC (Know Your Customer) is the process of a business being forced to identify and verify the identity of its clients, and to share this information with a government.
The term is also used to refer to the bank regulation which governs these activities.
Read more: [AML/KYC Information](/why-wasabi/TransactionSurveillanceCompanies.md#aml-kyc-information)
:::

:::details
### Lurking Wife Mode (LWM)

> Lurking Wife Mode is a Wasabi feature that hides sensitive and critical information on the wallet itself.

Reviewer: ```suggestion
Lurking Wife Mode is a Wasabi feature that hides sensitive and critical information on the wallet itself, which is useful for screenshots.
```

Read more: [Lurking Wife Mode](/using-wasabi/LurkingWifeMode.md)
:::

:::details
### Pay to EndPoint (P2EP)

> Pay to EndPoint simplifies the sending of bitcoins with more security and privacy than Pay to IP (P2IP).

Reviewer: ```suggestion
Pay to EndPoint is sending of bitcoins where the receiver adds one of his own coins as input for a two party CoinJoin.
```

I've never heard of P2IP?

:::

:::details
### Taint

Taint is equivalent to the 'trail' that a Bitcoin transaction leaves during the course of its journey.

> The taint analysis of a Bitcoin transaction evaluates the association between an address and the previous addresses involved in transactions.

Reviewer: ```suggestion
The taint analysis of a Bitcoin transaction evaluates the association between an address involved in the chain of transactions.
```

> The higher the number of taint, the stronger the connection between the two addresses.

Reviewer: ```suggestion

```
